pkgs/tools/filesystems/dwarfs/default.nix

```diff
@@ -1,50 +1,54 @@
-{ lib
-, fetchFromGitHub
-, stdenv
-, substituteAll
-
-, bison
-, boost
-, cmake
-, double-conversion
-, fmt
-, fuse3
-, glog
-, gtest
-, jemalloc
-, libarchive
-, libevent
-, libunwind
-, lz4
-, openssl
-, pkg-config
-, ronn
-, xxHash
-, utf8cpp
-, zstd
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  substituteAll,
+  bison,
+  boost,
+  cmake,
+  double-conversion,
+  fmt,
+  fuse3,
+  glog,
+  gtest,
+  jemalloc,
+  libarchive,
+  libevent,
+  libunwind,
+  lz4,
+  openssl,
+  pkg-config,
+  ronn,
+  xxHash,
+  utf8cpp,
+  zstd,
 }:
-
-stdenv.mkDerivation rec {
+let
   pname = "dwarfs";
-  version = "0.7.5";
-
+  version = "0.9.9";
+in
+stdenv.mkDerivation {
+  inherit pname version;
   src = fetchFromGitHub {
     owner = "mhx";
     repo = "dwarfs";
-    rev = "v${version}";
+    rev = "refs/tags/v${version}";
     fetchSubmodules = true;
     hash = "sha256-Zzm2SaFR31TBBMDfgJulVbqsJBh1He2wBFzHRC/c5vg=";
```

Comment:
The `version` was bumped but the `hash` is unchanged.

So this is really building the old version!